components/compass-runtime-agent/internal/compass/director/query.go

```diff
@@ -73,25 +73,17 @@ func bundlesData() string {
 		instanceAuthRequestInputSchema
 		apiDefinitions {%s}
 		eventDefinitions {%s}
-		documents {%s}
 		defaultInstanceAuth {%s}
```

Comment:
If we don't query for API documents here we can remove its processing code from tests.